### ISSUE TITLE
Fatal error: Declaration of LeKoala\Mailgun\MailgunSwiftTransport::send(Swift_Mime_SimpleMessage $message, &$failedRecipients = NULL) must be compatible with Swift_Transport::send(Swift_Mime_Message $message, &$failedRecipients = NULL) in /var/www/html/vendor/lekoala/silverstripe-mailgun/src/MailgunSwiftTransport.php on line 100

### ISSUE THREAD

**Issue by samandeggs** [lekoala/silverstripe-mailgun#8]
Hey - getting:

```
Fatal error: Declaration of LeKoala\Mailgun\MailgunSwiftTransport::send(Swift_Mime_SimpleMessage $message, &$failedRecipients = NULL) must be compatible with Swift_Transport::send(Swift_Mime_Message $message, &$failedRecipients = NULL) in /var/www/html/vendor/lekoala/silverstripe-mailgun/src/MailgunSwiftTransport.php on line 100
```

Presuming a mismatch of swiftmailer versus the expectations of this repo. Do you know what version of Swiftmailer this is guaranteed to work with? Otherwise I may have to roll SilverStripe back. Running 4.7.3.

**Comment by samandeggs** [comment]
Downgrading to `1.0.0` fixes it for now.

**Comment by lekoala** [comment]
yes, or upgrading to latest version of silverstripe^^

**Comment by plumduffer** [comment]
This is happening again with the latest version of SilverStripe 4.10 and plugin version 1.1.2

**Comment by lekoala** [comment]
@plumduffer it seems that 1.1.2 pr actually created the issue. I have reverted the changes and published 1.1.3

**Comment by plumduffer** [comment]
> @plumduffer it seems that 1.1.2 pr actually created the issue. I have reverted the changes and published 1.1.3

Super fast response, thank you!
